nntrainer/tensor/cpu_backend/fallback/fallback.h

```diff
@@ -21,6 +21,17 @@
 #include <stdexcept>
 #include <tensor_dim.h>
 
+#if defined(_WIN32)
+#ifdef max
+#undef max
+#undef min
+#endif
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+#include <windows.h>
+#endif
```

Comment:
The same Windows-related preprocessor code is repeated in multiple header files.
Would you be able to make a header file?
```
/* windows_compat.h */
#ifndef __WINDOWS_COMPAT_H__
#define __WINDOWS_COMPAT_H__

#if defined(_WIN32)
#ifdef max
#undef max
#undef min
#endif
#ifndef NOMINMAX
#define NOMINMAX
#endif
#include <windows.h>
#endif

#endif // __WINDOWS_COMPAT_H__
```
And use.
```
#include "windows_compat.h"
```